setup.cfg

```diff
@@ -34,7 +34,7 @@ install_requires =
     voila>=0.3.5
     pyyaml>=5.4.1
     specutils>=1.7.0
-    specreduce>=1.0.0,!=1.1.0
+    specreduce==1.1.0
```

Comment:
What's your motivation for pinning instead of setting 1.1.0 as the minimum?

Reply:
to avoid the situation we ran into yesterday.  Until we enforce deprecating API breaks in specreduce development, we need to ensure that an existing build of jdaviz won't be broken by a new release of specreduce.  We could probably allow for bugfixes though, with `specreduce>=1.1.0,<1.2.0`?

Reply:
I like that suggestion. I think `~1.1.0` means the same thing if we'd like to practice our semver-style shortcuts.

Reply:
I actually think that `~=1.1` would include `1.2` since it doesn't consider minor version bumps as breaking (https://peps.python.org/pep-0440/#compatible-release).  For now I'll just do it the verbose way to be safe, and hopefully we can loosen up the pin down the road.